package.json

```diff
@@ -40,13 +40,16 @@
     "webpack-dev-server": "^4.15.1"
   },
   "dependencies": {
+    "antd": "^5.12.8",
```

Comment:
I think the tabs can be done with [semantic-ui](https://semantic-ui.com/modules/tab.html) which is already added to the project.